lib/core/jtrace.c

```diff
@@ -550,6 +742,275 @@ j_trace_enter(gchar const* name, gchar const* format, ...)
 		G_UNLOCK(j_trace_echo);
 	}
 
+	if (j_trace_flags & J_TRACE_ACCESS)
```

Comment:
OTF and ECHO tarce level acquire a lock at this point. Is it not necessary for ACCESS level?